xbmc/settings/lib/SettingDefinitions.h

```diff
@@ -64,6 +64,7 @@
 #define SETTING_XML_ATTR_SETTING      "setting"
 #define SETTING_XML_ATTR_BEFORE       "before"
 #define SETTING_XML_ATTR_AFTER        "after"
+#define SETTING_XML_ATTR_HIDE         "hide"
```

Comment:
Becomes changed, is thought to hide separator and is a bit to short from name to see this.